frontend/src/app/modules/home/issues/issues.component.ts

```diff
@@ -43,11 +48,18 @@ export class IssuesComponent extends BaseComponent implements OnInit {
         event.originalEvent.stopPropagation();
     }
 
+    private showPaginator() {
+        if (this.length > 10) {
+            this.paginators = (this.paginators === false);
+        }
```

Comment:
Perhaps this if statement can be simplified.
```suggestion
        this.paginators = this.length > 10;
```
What do you think about this?

Reply:
Also, I think we should get rid of the magic number such as 10. Why don't we introduce a constant, like itemsPerPage, rowsPerPage, or pageSize.

Reply:
I'm sorry lint fix took my code and framed it